src/Microsoft.Performance.SDK.Runtime/Extensibility/BaseSourceProcessingSession.cs

```diff
@@ -1,16 +1,16 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System;
```

Comment:
I think we're deciding to keep System imports first. You can change this behavior in VS Options under Text Editor -> C# -> Advanced